### PR TITLE
TransactionRecord Parity

### PR DESCRIPTION
**Pull request by danielakhterov**

### CODE REVIEW

File: src/TransactionReceipt.ts [hunk 1]
```diff
@@ -5,26 +5,50 @@ import { FileId } from "./file/FileId";
...
-export interface TransactionReceipt {
-    status: Status;
-    accountId: AccountId | null;
-    fileId: FileId | null;
-    contractId: ContractId | null;
-    exchangeRateSet: ExchangeRateSet | null;
-}
+export class TransactionReceipt {
+    public readonly status: Status;
+    private accountId: AccountId | null;
```
Comment (mehcode) [inline src/TransactionReceipt.ts]: Use underscore prefixes for private fields

File: src/TransactionReceipt.ts [hunk 2]
```diff
@@ -5,26 +5,50 @@ import { FileId } from "./file/FileId";
...
-            null,
+    public getContractId(): ContractId {
+        return this.contractId!;
+    }
 
-        exchangeRateSet: receipt.hasExchangerate() ?
-            exchangeRateSetToSdk(receipt.getExchangerate()!) :
-            null
-    };
+    public static fromProto(receipt: ProtoTransactionReceipt): TransactionReceipt {
```
Comment (mehcode) [inline src/TransactionReceipt.ts]: Should have an underscore prefix

File: src/Time.ts [hunk 3]
```diff
@@ -10,4 +12,8 @@ export class Time {
     public asDate(): Date {
         return new Date(this.seconds * 1000 + Math.floor(this.nanos / 1_000_000));
     }
+
+    public static fromProto(timestamp: ProtoTimestamp): Time {
```
Comment (mehcode) [inline src/Time.ts]: Underscore this too

File: src/TransactionReceipt.ts (hunk 1 above)
Comment (danielakhterov) [reply src/TransactionReceipt.ts]: Resolved in #147

**Pull request merged**